.gitignore

```diff
@@ -30,3 +30,6 @@ clap-rs.iml
 test-results.test
 expanded.rs
 clap_derive/expanded.rs
+
+# ctags
+**/*.vi
```

Comment:
Is this necessary? I think you could just put this into your global `.config/git/ignore` right?

Reply:
@kbknapp's setup uses it. Harmless. I always prefer to allow the core maintainers to add stuff to gitignore.